### PR TITLE
Magic/sorter

### PR DESCRIPTION
**Pull request by gkaretka**
Adding main files.

### CODE REVIEW

**Comment by pepol** [comment]
@magic-sudo Sorter doesn't have to be a class. Namespace would be much better suited for this. Also, since it doesn't really do sorting, I would probably rename it to something else. We can rename the repository to `utilities` (for example) later and add more utility functions here as well.

File: sorter.h [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#ifndef sorter_h
+#define SORTER_H
+
+namespace Sorter {
+    void sort(int s[12][2]);
+    void swap(int *a, int *b);
+}
+
+#endif // sorter_h
```
Comment (pepol) [inline sorter.h]: `// SORTER_H` and please put newline here (the diff shows it's missing).

File: sorter.h [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#ifndef sorter_h
```
Comment (pepol) [inline sorter.h]: `sorter_h` should be `SORTER_H`.

File: sorter.h [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+#define SORTER_H
+
+namespace Sorter {
```
Comment (pepol) [inline sorter.h]: Namespace names should be all lowercase.

File: sorter.h (hunk 2 above)
Comment (gkaretka) [inline sorter.h]: done

File: sorter.h (hunk 1 above)
Comment (gkaretka) [inline sorter.h]: done

File: example/example.ino [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#include <sorter.h>
+
+Sorter s;
```
Comment (pepol) [inline example/example.ino]: This won't compile now. Please fix it in the whole file.

File: sorter.h (hunk 3 above)
Comment (gkaretka) [inline sorter.h]: done

File: keywords.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Sorter	KEYWORD1
```
Comment (pepol) [inline keywords.txt]: There is no longer a `Sorter` class.

**Comment by pepol** [comment]
@magic-sudo looks good, please resolve all comments and test this.

File: example/example.ino (hunk 4 above)
Comment (gkaretka) [inline example/example.ino]: done

File: keywords.txt (hunk 5 above)
Comment (gkaretka) [inline keywords.txt]: done

**Comment by pepol** [comment]
LGTM, please squash.

**Comment by pepol** [comment]
Thanks!

**Pull request merged**